src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,51 @@
  */
 
 public class Plane {
+    private int param1;
+    private int param2;
+    private int param3;
+    private int param4;
+    private int param5;
 
+    public Plane(PlaneBuilder planeBuilder) {
+
+    }
+
+    public class PlaneBuilder {
+
+        private int paramParam;
+
+        public PlaneBuilder(int param1) {
+            Plane.this.param1 = param1;
```

Comment:
should builder have any constructors?

Reply:
i did rewrite whole solution.
public static void main(String[] args) {
        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder();
        planeBuilder.setName("AirPlane");

        System.out.println();//blaBla business logic
        System.out.println();//blaBla business logic

        Plane plane = planeBuilder
                .setHeight(1)
                .setModel("Model")
                .setWeight(2)
                .setAge(999)
                .build();
    }
now it work